test/prism_regression/def_block_param.rb

```diff
@@ -1,9 +1,5 @@
 # typed: false
 
-def foo(&blk); end
```

Comment:
This name was easy to confuse with `<blk>` (`core::Names::blkArg()`), so I opted to change it.